modules/auxiliary/admin/mysql/mysql_sql.rb

```diff
@@ -14,7 +15,8 @@ def initialize(info = {})
           against a MySQL instance given the appropriate credentials.
       },
       'Author'		=> [ 'Bernardo Damele A. G. <bernardo.damele[at]gmail.com>' ],
-      'License'		=> MSF_LICENSE
+      'License'		=> MSF_LICENSE,
+      'SessionTypes' => %w[MySQL]
```

Comment:
Just a reminder to check if the `SessionType` values are compared in a case-sensitive or insensitive manner.

Reply:
This is going to be handled as part of another ticket now.